app/src/main/java/com/eveningoutpost/dexdrip/Services/WixelReader.java

```diff
@@ -123,24 +120,22 @@ public static boolean almostEquals( TransmitterRawData e1, TransmitterRawData e2
         return merged;
     }
 
-    public static List<TransmitterRawData> MergeLists(List <List<TransmitterRawData>> allTransmitterRawData)
-    {
-        List<TransmitterRawData> MergedList;
-        MergedList = allTransmitterRawData.remove(0);
+    public static List<TransmitterRawData> mergeLists(List<List<TransmitterRawData>> allTransmitterRawData) {
+        List<TransmitterRawData> mergedList;
+        mergedList = allTransmitterRawData.remove(0);
         for (List<TransmitterRawData> it : allTransmitterRawData) {
-            MergedList = Merge2Lists(MergedList, it);
+            mergedList = merge2Lists(mergedList, it);
         }
 
-        return MergedList;
+        return mergedList;
     }
 
-    public static List<TransmitterRawData> ReadHost(String hostAndIp, int numberOfRecords)
-    {
+    public static List<TransmitterRawData> readHost(String hostAndIp, int numberOfRecords) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method readHost() has an NPath complexity of 24](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1313443989/issues/source?bid=2735053&fileBranchId=2735067#l133)